tests/integration_tests.rs

```diff
@@ -531,6 +586,22 @@ async fn test_wasi_node_should_verify() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_experimental_wasi_http_pod() -> anyhow::Result<()> {
+    const EXPECTED_POD_LOG: &str = r#"{"args":{},"data":"I'm not superstitious, but I am a little stitious.","files":{},"form":{},"headers":{"accept":"*/*","content-length":"50","content-type":"text/plain","host":"postman-echo.com","x-amzn-trace-id":"Root=1-60f9e64c-36256b4878dc640c5af75f7d","x-forwarded-port":"443","x-forwarded-proto":"https"},"json":null,"url":"https://postman-echo.com/post"}"#;
+
+    let test_ns = "wasi-e2e-experimental-http";
+    let (client, pods, mut resource_manager) = set_up_test(test_ns).await?;
+
+    create_experimental_wasi_http_pod(client.clone(), &pods, &mut resource_manager).await?;
+
+    assert::pod_log_equals(&pods, EXPERIMENTAL_WASI_HTTP_POD, EXPECTED_POD_LOG).await?;
```

Comment:
It think it is a bit safer to just do `pod_log_contains` here in case any of the other output changes (which I believe is why the e2e tests are failing)